cloudsql-mysql-plugin/src/e2e-test/java/io/cdap/plugin/CloudMySql/runners/TestRunnerRequired.java

```diff
@@ -0,0 +1,4 @@
+package io.cdap.plugin.CloudMySql.runners;
+
```

Comment:
If you are creating this class use it properly define the cucumber options and use the tag in the tests which can be run as required tests.

Reply:
resolved

Reply:
Add the link of commit whenever you mark a comment as resolved.